src/polyfill.ts

```diff
@@ -1,16 +0,0 @@
-import 'core-js/es/math';
-import 'core-js/es/number';
-/* eslint no-proto: 0 */
-const testObject = {} as any;
-
-if (!(Object.setPrototypeOf || testObject.__proto__)) {
-  const nativeGetPrototypeOf = Object.getPrototypeOf;
-
-  Object.getPrototypeOf = object => {
```

Comment:
I grep for `getPrototypeOf` and seems it is unused (as of latest `master` code).

So I just removed this code too.